src/task/Epic.java

```diff
@@ -1,9 +1,11 @@
 package task;
 
+import java.time.LocalDateTime;
 import java.util.ArrayList;
 
 public class Epic extends Task {
     private ArrayList<Integer> subTaskIds;
```

Comment:
Нужно обозначать интерфейс
```suggestion
    private List<Integer> subTaskIds;
```